2025-07-18 10:48:39,023 - asyncio - DEBUG - Using proactor: IocpProactor
2025-07-18 10:48:39,023 - httpx - DEBUG - load_ssl_context verify=True cert=None trust_env=True http2=False
2025-07-18 10:48:39,031 - httpx - DEBUG - load_verify_locations cafile='c:\\Users\\Rachid Aitali\\OneDrive - 4JM Solutions Limited\\Desktop\\SniperX-V3-Best-Version-at-now\\SniperX-V3-Best-Version-at-now\\.venv\\lib\\site-packages\\certifi\\cacert.pem'
2025-07-18 10:48:39,049 - httpx - DEBUG - load_ssl_context verify=True cert=None trust_env=True http2=False
2025-07-18 10:48:39,051 - httpx - DEBUG - load_verify_locations cafile='c:\\Users\\Rachid Aitali\\OneDrive - 4JM Solutions Limited\\Desktop\\SniperX-V3-Best-Version-at-now\\SniperX-V3-Best-Version-at-now\\.venv\\lib\\site-packages\\certifi\\cacert.pem'
2025-07-18 10:48:39,094 - __main__ - INFO - Telegram Manager Bot started. Send /menu to interact.
2025-07-18 10:48:39,094 - telegram.ext.ExtBot - DEBUG - Entering: send_message
2025-07-18 10:48:39,112 - httpcore.connection - DEBUG - connect_tcp.started host='api.telegram.org' port=443 local_address=None timeout=5.0 socket_options=None
2025-07-18 10:48:39,160 - httpcore.connection - DEBUG - connect_tcp.complete return_value=<httpcore._backends.anyio.AnyIOStream object at 0x000001BBAF48F3D0>
2025-07-18 10:48:39,160 - httpcore.connection - DEBUG - start_tls.started ssl_context=<ssl.SSLContext object at 0x000001BBAF444540> server_hostname='api.telegram.org' timeout=5.0
2025-07-18 10:48:39,264 - httpcore.connection - DEBUG - start_tls.failed exception=ConnectError(BrokenResourceError())
2025-07-18 10:48:39,264 - __main__ - ERROR - Could not send startup message to TELEGRAM_CHAT_ID 7049055170: httpx.ConnectError: 
2025-07-18 10:48:39,264 - telegram.ext.ExtBot - DEBUG - Entering: get_me
2025-07-18 10:48:39,269 - httpcore.connection - DEBUG - connect_tcp.started host='api.telegram.org' port=443 local_address=None timeout=5.0 socket_options=None
2025-07-18 10:48:39,279 - httpcore.connection - DEBUG - connect_tcp.complete return_value=<httpcore._backends.anyio.AnyIOStream object at 0x000001BBB04ACFA0>
2025-07-18 10:48:39,279 - httpcore.connection - DEBUG - start_tls.started ssl_context=<ssl.SSLContext object at 0x000001BBAF444540> server_hostname='api.telegram.org' timeout=5.0
2025-07-18 10:48:39,342 - httpcore.connection - DEBUG - start_tls.failed exception=ConnectError(BrokenResourceError())
2025-07-18 10:48:39,342 - telegram.ext.Application - DEBUG - This Application is already shut down. Returning.
2025-07-18 10:48:39,343 - __main__ - ERROR - Error in polling: httpx.ConnectError: 
